content/applications/general/email_communication/faq.rst

```diff
@@ -128,11 +132,10 @@ Email is sent late
 Email campaigns send at a scheduled time, using a delay pre-programed in the database. Odoo uses a
 delayed task to send emails that are considered "not urgent" (newsletter formats, such as: mass
 mailing, marketing automation, and events). The system utility **cron** can be used to schedule
-programs to run automatically at predetermined intervals. Odoo uses that policy in order to avoid
-cluttering the mail servers and, instead, prioritizes individual communication. This **cron** is
-called :guilabel:`Mail: Email Queue Manager`, and can be accessed in :ref:`developer mode
-<developer-mode>` by going to :menuselection:`Settings app --> Technical menu --> Automation -->
-Scheduled Actions`.
+programs to run automatically at predetermined intervals. Odoo uses that policy to avoid cluttering
+the mail servers and, instead, prioritizes individual communication. This **cron** is called
+:guilabel:`Mail: Email Queue Manager`, and can be accessed in :ref:`developer mode <developer-mode>`
+by going to :menuselection:`Settings app --> Technical menu --> Automation --> Scheduled Actions`.
```

Comment:
```suggestion
by going to :menuselection:`Settings app --> Technical menu --> Automation: Scheduled Actions`.
```
based on the feedback we discussed during the latest Writer's Workshop meeting, I *believe* this is how we decided to format such a direction moving forward